docs/utils/index.rst

```diff
@@ -17,16 +17,17 @@ means:
 
  * the many kinds of warnings and exceptions you may (hopefully not!)
    encounter while working with PlasmaPy, such as
-   `plasmapy.utils.RelativityWarning` or `plasmapy.utils.PhysicsError`.
+   `~plasmapy.utils.RelativityWarning` or `~plasmapy.utils.PhysicsError`.
```

Comment:
```suggestion
   `~plasmapy.utils.exceptions.RelativityWarning` or `~plasmapy.utils.exceptions.PhysicsError`.
```

Since we do not `automodapi` or `automodule` `plasmapy.utils`,  nothing defined in the `__all__` of `plasmapy/utils/__init__.py` is documented; thus, there is no stub file for things like `plasmapy.utils.RelativityWarning`.